src/loop_libraries/loop_library.cpp

```diff
@@ -59,11 +60,19 @@ void Loop_library::set_default()
 {
    Loop_library::lib_ = std::make_unique<looplibrary::Softsusy>();
    Loop_library::type_ = Loop_library::Library::Softsusy;
+   VERBOSE_MSG("Enabling softsusy.");
 }
 
 void Loop_library::set(int new_type)
 {
    if (Loop_library::type_ == Loop_library::Library::Undefined) {
+      if (new_type == -1) {
+         if (const char* flag = std::getenv("FLEXIBLESUSY_LOOP_LIBRARY")) {
+            new_type = std::atoi(flag);
```

Comment:
Maybe we could wrap above line in a try-catch blocks? And print something meaningful if the variable is not set/contains not a numer.

Reply:
If it is not a number it returns 0, i.e. `softsusy`

Reply:
And exactly one line below it is printed, in case of `--enable-verbose`.

Reply:
Ah, ok, so this nicely fits together. :)